test/parallel/test-util-log.js

```diff
@@ -62,4 +62,5 @@ tests.forEach(function(test) {
 
 assert.strictEqual(process.stdout.writeTimes, tests.length);
 
-restoreStdout();
+common.restoreStdout();
+common.restoreStderr();
```

Comment:
nit: newline at the end of the file

Reply:
(I added it.)

Reply:
@Trott this has to be: `restoreStdout()` and `restoreStderr()`. So the `common.` part has to be removed on both. `restoreStderr` has to be destructured along the other parts.

Reply:
@BridgeAR I didn't author any code. All I did here was add a missing newline at end of file. (I should have been more explicit about that in my previous comment.)